### ISSUE TITLE
Need to output TOA reflectance mosaics for band 4 and potentially all bands

### ISSUE THREAD

**Issue by BaptisteVandecrux** [GEUS-SICE/SICE#49]
In the new version of the algorithm, we need bands 1 and 4 for impurity retrieval.
Then bands 17 and 21 for snow properties.

Right now in the S3_fast.xml only process selected bands to save time.

For the integration of BBA that will be implemented (again) shortly, we will need all the bands.

**Comment by AdrienWehrle** [comment]
So let's just use [S3.xml](https://github.com/GEUS-SICE/SICE/blob/master/S3.xml) by default?

**Comment by BaptisteVandecrux** [comment]
yes, at least that should be done from the moment we switch [pySICEv2.0](https://github.com/GEUS-SICE/pySICE/tree/pySICEv2) that implements the new algorithm from Alex

**Comment by AdrienWehrle** [comment]
Sounds good! pySICEv2.0, exciting!! :partying_face:
